src/main/java/net/glowstone/block/itemtype/ItemType.java

```diff
@@ -164,10 +166,67 @@ public final String toString() {
         return getClass().getSimpleName() + "{" + getId() + " -> " + getMaterial() + "}";
     }
     
+    //Code for wearable items
+    private ItemWearablePosition wearablePosition = ItemWearablePosition.NON_WEARABLE;
+    private int armorPoints = 0;
+    private boolean fastEquip = false;
+    
     public ItemWearablePosition getWearablePosition() {
         return wearablePosition;
     }
     public void setWearablePosition(ItemWearablePosition position) {
```

Comment:
This is invalid for most items. Keep the subclass for wearable items. Is there a reason the class was removed? And what were the use cases of the class and these methods?